### PR TITLE
build manylinux1 wheel for Linux python2.7

### PR DESCRIPTION
**Pull request by anthrotype**
fixes https://github.com/adobe-type-tools/afdko/issues/251

### CODE REVIEW

**Comment by anthrotype** [comment]
Looks like this additional "Linux-deploy" build that I added to the Travis matrix does not run from a PR, but only from pushes to master (i.e. once this PR branch is merged), since I configured the same as the other "OSX-deploy" build (`if: type == push`).

Anyway, I tested running the build_manylinux_wheel.sh script locally with the same docker image and seems to work. I get a afdko-...-cp27-cp27mu-manylinux1_x86_64.whl in the dist/ folder, ready to be uploaded with twine.

This proposed setup basically will run the PR builds from the default ubuntu Travis image, but will build the linux wheel for deployment from a PyPA manylinux1 docker container.

Maybe it would make more sense to build the manylinux wheel everytime, not just in the before_deploy step, and install that in the ubuntu Travis python (to verify it was built correctly) and run the tests against that?

File: .travis/before_deploy.sh [hunk 1]
```diff
@@ -7,8 +7,7 @@ set -x
 # Travis runs the `before_deploy` stage before each deployment, but
 # we only want to build them once, as we want to use the same
 # files for both Github and PyPI
-if $(ls ./dist/afdko*.zip > /dev/null 2>&1) && \
```
Comment (anthrotype) [inline .travis/before_deploy.sh]: (btw, the check for .zip was wrong, since we are using the default .tar.gz format for the sdist)

**Comment by miguelsousa** [comment]
> Maybe it would make more sense to build the manylinux wheel everytime, not just in the before_deploy step, and install that in the ubuntu Travis python (to verify it was built correctly) and run the tests against that?

Sounds good to me.

**Comment by miguelsousa** [comment]
@anthrotype I think something didn't work, because there's no additional wheel associated with `2.7.0b2` https://pypi.org/project/afdko/2.7.0b2/#files

**Comment by miguelsousa** [comment]
oh, here's the error https://travis-ci.org/adobe-type-tools/afdko/jobs/368827801

**Comment by anthrotype** [comment]
It thinks it is ruby.. probably missing some global language setting. I’ll look into it tomorrow

**Pull request merged**